src/styles/base.js

```diff
@@ -0,0 +1,10 @@
+import { StyleSheet } from 'react-native';
```

Comment:
Instead of defining a folder with styles, you could create a file named ```styles.js``` inside each component folders, and define here these styles.

Reply:
See #6